### ISSUE TITLE
[BUG] Pyradio doesn't remember warning message was already shown

### ISSUE THREAD

**Issue by iodream** [coderholic/pyradio#239]
**Describe the bug**

After a recent update Pyradio started warning me about Themes being disabled because "Curses (the library this program is based on), will not display colors correctly in this terminal" with an option to dismiss this warning forever if you press x . Problem is if I press X , and restart pyradio the warning is there again every time. This was not the case in earlier versions. This also happens with an empty pyradio config folder. In the attached log file I've started pyradio, dismissed the warning box and restarted pyradio. 

Pyradio installed from the AUR.

**To Reproduce**
1. Open pyradio
2. get warning about themes disabled 
3. Press X to never show again
4. reopen Pyradio and get warning again

**Expected behavior**
No warning after pressing X to dismiss forever

**System (please complete the following information):**
 - OS: Arch linux
 - **Python** version: 3.11.8
 - **PyRadio** Version: 0.9.3.3-1
 - Player used: mpv

[pyradio.log](https://github.com/coderholic/pyradio/files/15050391/pyradio.log)

![image](https://github.com/coderholic/pyradio/assets/42720874/55591582-b14d-444f-9f0c-353ec1be6aa7)

**Comment by FarParticul** [comment]
look in this question #235  only in konsole this problem will occur and you will have to use another terminal

**Comment by FarParticul** [comment]
The solution was that I changed the terminal to kitty, and I highly recommend you try it

**Comment by iodream** [comment]
> look in this question #235 only in konsole this problem will occur and you will have to use another terminal

Yeah  I  know only Konsole is affected. Just to be clear, I'm ok with themes disabled, I'd just like the error to go away. 

And in previous versions it wouldn't bother me every time. If the developer says this isn't a bug or can't be fixed then I understand.

**Comment by FarParticul** [comment]
As I understand it, the problem is in curses and that’s why it doesn’t work

**Comment by s-n-g** [comment]
Hi guys!

This is a surely regression, due to the changes made to the way the config file is saved.
I'm looking into it

For the time being, to disable the annoying message, execute:

```
echo "show_no_themes_message = False" >> ~/.config/pyradio/config
```

**Comment by s-n-g** [comment]
fixed in **0.9.3.4**
